### ISSUE TITLE
Research Kotlin for backend

### ISSUE THREAD

**Issue by hsjoberg** [hsjoberg/blixt-wallet#110]

**Comment by sesam** [comment]
Is the idea to move features from JS/TS to kotlin? How does this affect iOS app version?

**Comment by hsjoberg** [comment]
@sesam No, the idea was to replace the current Java backend (the code that is talking with lnd) with Kotlin instead.
To be specific, we have LndMobileService.java, which is an Android Service that is talking with lnd.

It doesn't affect the iOS version, it's using Swift and is pretty well organized.
